CHANGELOG.md

```diff
@@ -5,42 +5,45 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ## [Unreleased][unreleased]
 
 ### Added 
-- Support 10.3 tokens
-- Add "?ping" for easier testing
-- (DotNet) Support Windows authentication
-- Support federated service
+- Support for 10.3 tokens
+- Added "?ping" for easier testing
+- (DotNet) Support for Windows authentication
+- Support for federated services
 
 ### Fixed
-- Better referrer handling
-- Support ? in password
-- Multiple redirects
-- Handle token expiration
-- Remove HTTP range-request Header
+- Better referer handling
+- Support for '?' in passwords
+- Handling of multiple redirects
+- Handling of token expiration
+
+### Removed
```

Comment:
The "fix" was to not return range-request header, since the proxy doesn't support it.  It's not like we "removed" a functionality that existed.  I think it belongs better under "Fixed" than "Removed" - but I'm open for discussion :)